mod/ping.php

```diff
@@ -195,13 +196,20 @@ function ping_init(App $a)
 			}
 		}
 
-		$ev = qu("SELECT count(`event`.`id`) AS total, type, start, adjust FROM `event`
-			WHERE `event`.`uid` = %d AND `start` < '%s' AND `finish` > '%s' and `ignore` = 0
-			ORDER BY `start` ASC ",
-			intval(local_user()),
-			dbesc(datetime_convert('UTC', 'UTC', 'now + 7 days')),
-			dbesc(datetime_convert('UTC', 'UTC', 'now'))
-		);
+		$cachekey = "ping_init:".local_user();
+		$ev = Cache::get($cachekey);
```

Comment:
Some note: At the present time we don't make use of the event notifier in any of our themes. So I'm fine with this change. But in general it doesn't seem to me a good solution to cache stuff over an hour where the user would expect up-to-date data.